lib/temporal/workflow/context_validators.rb

```diff
@@ -0,0 +1,24 @@
+
+module Temporal
+  class Workflow
+    class Context
+      module Private
+        # Shared between Context and LocalWorkflowContext so we can do the same validations in test and production.
+        module Validators
```

Comment:
Can you please match the file name/path to the module name and namespace (like we do with all the other classes/modules)?

Reply:
I guess the house style is not to use `Private` module.